node_module error with `v0.1.2` docker build

When following the readme to run the latest build with docker-compose, the container will fail to start with this error:

```
~/Documents/test/resonance 
› docker compose up -d
[+] up 21/21
 ✔ Image ghcr.io/jordonet/resonance:latest Pulled                                                                                            11.6s 
 ✔ Network resonance_default Created                                                                                                          0.2s 
 ✔ Container resonance Created                                                                                                                0.0s 

~/Documents/test/resonance 
› docker ps
CONTAINER ID   IMAGE                               COMMAND                  CREATED         STATUS                        PORTS     NAMES
7cc4e61e858c   ghcr.io/jordonet/resonance:latest   "docker-entrypoint.sh"   4 seconds ago   Restarting (1) 1 second ago             resonance

~/Documents/test/resonance 
› docker logs -f resonance 
node:internal/modules/cjs/loader:1424
  throw err;
  ^

Error: Cannot find module 'winston'
Require stack:
- /app/dist/config/logger.js
- /app/dist/server.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1421:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1059:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1064:22)
    at Module._load (node:internal/modules/cjs/loader:1227:37)
    at TracingChannel.traceSync (node:diagnostics_channel:328:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:245:24)
    at Module.require (node:internal/modules/cjs/loader:1504:12)
    at require (node:internal/modules/helpers:152:16)
    at Object.<anonymous> (/app/dist/config/logger.js:6:19)
    at Module._compile (node:internal/modules/cjs/loader:1761:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/dist/config/logger.js', '/app/dist/server.js' ]
}

Node.js v24.13.0
node:internal/modules/cjs/loader:1424
  throw err;
  ^
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -98,7 +98,7 @@ RUN cd /app/node_modules/.pnpm/sqlite3@5.1.7/node_modules/sqlite3 && \
     echo "SQLite3 production build complete"
 
 # Copy built server from builder
-COPY --from=server-builder /build/server/dist ./dist
+COPY --from=server-builder /build/server/dist ./server/dist
 
 # Copy built ui to static directory
 COPY --from=ui-builder /build/ui/dist ./static

--- a/docker-entrypoint.sh
+++ b/docker-entrypoint.sh
@@ -20,4 +20,4 @@ if [ "$(id -u)" = "0" ]; then
 fi
 
 # Now running as resonance user - start the application
-exec node dist/server.js
+exec node server/dist/server.js

